### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by hyoong**
Added figure numbering using \begin{figure}[H] .... \caption{<text>} ... \end{figure}
The [H] is need to ensure imaged will be captioned at the exact location in your tex.

### CODE REVIEW

File: docs/report/main.tex [hunk 1]
```diff
@@ -404,7 +425,7 @@ \subsection{Gates and Operators}
 \vspace{5mm}
 
 \noindent
-The phase shift gate modifies the phase of the quantum state and is equivalent to tracing a horizontal circle  on the Bloch Sphere(Refer to page 3) by $\varphi$ radians. 
+The phase shift gate modifies the phase of the quantum state and is equivalent to tracing a horizontal circle  on the Bloch Sphere (Refer to Figure 1 on on page 3) by $\varphi$ radians\cite{noauthor_quantum_nodate}.
```
Comment (Tiernan8r) [inline docs/report/main.tex:428]: If you add labels to the figure captions, you can use the `\ref` tag to auto number your figure references.

Comment (hyoong) [reply docs/report/main.tex:428]: Good shout bro. I have added the label in now. I will create PR later when I add more in the report. I am going to write some bit for QFT.

Comment (Tiernan8r) [reply docs/report/main.tex:428]: I'm working on splitting the figure numbering in the TikZ charts you have into fig 1a/Fig 1b. So could do it as part of a PR for that if you want.

**Review by Tiernan8r — APPROVED** [review]

File: docs/report/main.tex (hunk 1 above)
Comment (hyoong) [reply docs/report/main.tex:428]: sure Tiernan, you are welcome to do that. thanks!

Comment (Tiernan8r) [reply docs/report/main.tex:428]: https://github.com/Tiernan8r/quantum_computing_project/pull/80 should do it

**Pull request merged**